### ISSUE TITLE
Differences to google/omaha

### ISSUE THREAD

**Issue by NtFreX** [Crystalnix/omaha#33]
Hey guys

I'm using the crystalnix/omaha-server and i'm currently deciding between google/omaha and crystalnix/omaha.

The commit ratio is in the google repo much higher but I saw that crystalnix/omaha partly supports staging (channel tag).

Whats the difference between those repos? Are changes from google/omaha merged into crystalnix/omaha? Are there other code and functionallity changes?

**Comment by thekondr** [comment]
Hi xDrFred,

A long time ago Google pushed their current state of Omaha client and then they didn't update it for many years.

At Crystalnix we tried to resurrect the project. We fixed some build issues, added support for latest Windows systems and implemented the channel tag.

Then few months later Google released source code of their Omaha client once again (many years later since the previous update).

So basically google/omaha is more up-to-date and we are planning to merge their changes into our repository once we have time to do it.
